homestar-runtime/fixtures/settings.toml

```diff
@@ -6,3 +6,4 @@ process_collector_interval = 10
 [node.network]
 events_buffer_len = 1000
 websocket_port = 9999
+trusted_node_addresses = ["/ip4/127.0.0.1/tcp/9998/ws"]
```

Comment:
for the setting, I rather name these `node addresses`. We'll assume their trusted, but a comment/doc saying these will be trusted would be good.